test/e2e/pages/move-detail.js

```diff
@@ -189,6 +192,20 @@ class MoveDetailPage extends Page {
       }
     }
   }
+
+  async checkUpdateLink(category, exists = true) {
+    const selector = this.nodes.getUpdateLink(category)
+    await t.expect(selector.exists).eql(exists)
+  }
+
+  async checkNoUpdateLink(category) {
+    await this.checkUpdateLink(category, false)
+  }
+
+  async clickUpdateLink(category, exists = true) {
```

Comment:
I wonder if these are actually an abstraction too many. This function doesn't really share anything that isn't already shared. The selector is abstracted to one place using the page nodes and `t.click` feels already clear enough in a test on its own.